dts/bindings/reset/reset-controller.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Andrei-Edward Popa <andrei.popa105@yahoo.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Reset Controller
+
+include: base.yaml
```

Comment:
This should be included by the yaml file of the reset controllers, not by this file.